con-master/build.gradle

```diff
@@ -1,10 +1,11 @@
-jar{
+jar {
     version = '0.0.1-SNAPSHOT'
     baseName = 'con-master'
 }
 
 dependencies {
     compile project(":con-base")
+    compile('log4j:log4j:1.2.17')
```

Comment:
We are going to use log4j in whole project, so it would be better to set it in main build.gradle. Also, check and remove existing logging dependencies which we get through spring-boot-starter-logging.